torch/autograd/graph.py

```diff
@@ -657,9 +657,18 @@ def iter_graph(roots):
 
             yield node
 
-    def prehook(grad_output):
+    def fmt(t):
+        # Avoid circular import
+        from torch.testing._internal.common_utils import dtype_abbrs
+
+        if t is None:
+            return "None"
+        return f"{dtype_abbrs[t.dtype]}[{', '.join(map(str, t.shape))}]"
```

Comment:
Is that a usual format used in other places?

Reply:
Yeah, fx and logging tensor also use this format